docker/docker-entrypoint.sh

```diff
@@ -47,5 +47,8 @@ pip install -r requirements.txt
 
 wait-for-it.sh -h "${ADDR}" -p "${PORT}" -t 90 -- >&2 echo "Rocket.Chat is ready"
 
+>&2 echo "Checking if hubot is available"
+env PYTHONPATH="/root/rocketchat-tests-based-on-splinter/" ${PYTHON} /root/is_bot_online.py --username="${USERNAME}" --password="${PASSWORD}" --wait="120" --bot="meeseeks"
```

Comment:
The `wait` argument has a default value and it is 100 seconds. So we can not set it here.

Reply:
Use `"${BOT_NAME}"` instead of `meeseeks`.